imap_processing/tests/test_cli.py

```diff
@@ -35,6 +35,22 @@
 from imap_processing.spice import config as spice_config
 
 
+@pytest.fixture(autouse=True)
+def clear_spice_kernels():
+    """Fixture to clear SPICE kernels before each test."""
+    global_kernels = []
+    for i in range(spiceypy.ktotal("all")):
+        data = spiceypy.kdata(i, "all")
+        if data[1] == "META" or data[2] == "":
+            global_kernels.append(data[0])
+    try:
+        spiceypy.kclear()
+        yield
+    finally:
+        spiceypy.kclear()
+        spiceypy.furnsh(global_kernels)
```

Comment:
I think that this functionality is implemented in the `spiceypy.KernelPool()` context manager.
```suggestion
    with spiceypy.KernelPool([]):
        yield
```

Reply:
I had tried this but for some reason It wasnt working and I was assuming it was because of the empty list of kernels.

Reply:
Ah ok wait never mind I think I was doing it wrong.